apps/tangle-dapp/types/bridge.ts

```diff
@@ -44,3 +45,8 @@ export type BridgeTokenType = {
    */
   erc20TokenContractAddress?: Partial<Record<ChainId, HexString>>;
 };
+
+export enum BridgeWalletError {
+  MismatchEvm = 'wallet-mismatch-evm',
+  MismatchSubstrate = 'wallet-mismatch-substrate',
```

Comment:
I think we need to maintain consistency for the string enum values. I suggest changing them to all upper snake-case.

`'wallet-mismatch-evm' -> 'WALLET_MISMATCH_EVM'`.